decidim-admin/config/locales/en.yml

```diff
@@ -1115,28 +1115,33 @@ en:
           success: Taxonomy filter successfully destroyed.
         edit:
           back: Back
-          items: "%{count} items selected"
-          no_items: No items are available for this taxonomy. You can create items in the taxonomy configuration settings.
           title: Edit taxonomy filter
           update: Update taxonomy filter
         form:
-          select_taxonomy: Select a taxonomy
+          all: All
+          items: "%{count} items selected"
+          name_help: The title of the filter to be displayed in the frontend.
+          no_items: No items are available for this taxonomy. You can create items in the taxonomy configuration settings.
+          space_filter: Available as filter for all %{manifest_name}
         index:
           description: A taxonomy filter allows admins to sort and filter participatory spaces based on a taxonomy. For example, add a taxonomy filter to sort processes by geographical scope.
           empty: There are currently no taxonomy filters. Create a list of taxonomy filters here to sort and filter participatory spaces based on a taxonomy.
         new:
           back: Back
           create: Create taxonomy filter
-          items: "%{count} items available"
+          select_taxonomy: Select a taxonomy
           title: New taxonomy filter
         table:
           actions: Actions
-          components_count: Components count
+          components_count: Associated components
           confirm_destroy: 'Are you sure you want to delete the filter %{name}? NOTE: Resources that are using this filter will not be affected but they will lose the reference to this filter.'
-          count: Number of items
+          count: Number of items used
           destroy: Delete
           edit: Edit
-          name: Name
+          internal_name: Internal filter name
+          name: Public filter title
+          space_filter: Use to filter %{manifest_name}
```

Comment:
```suggestion
          space_filter: Used to filter %{manifest_name}
```

Reply:
done!
Btw, I the problem with the taxonomies in the review app is related to the fact that data was entered before we introduced changes in the database to set up cache counters in Rails. It should be fine now